rails/client/webpack.config.js

```diff
@@ -84,99 +90,102 @@ const config = {
       },
       {
         test: /\.(less)$/,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: [
-            {
-              loader: 'css-loader',
-              options: {
-                sourceMap: true,
-                importLoaders: 2,
-                localIdentName: '[name]_[local]_[hash:base64:5]',
-              },
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              sourceMap: true,
+              importLoaders: 2,
+              localIdentName: '[name]_[local]_[hash:base64:5]',
             },
-            {
-              loader: 'postcss-loader',
-              options: {
-                ident: 'postcss',
-                plugins() {
-                  return [
-                    Autoprefixer({
-                      browsers: [
-                        'ie >= 10',
-                        'last 2 versions',
-                        'last 2 android versions',
-                        'last 2 and_chr versions',
-                        'iOS >= 8',
-                      ],
-                    }),
-                    CustomProperties({
-                      warnings: false,
-                    }),
-                  ];
-                },
+          },
+          {
+            loader: 'postcss-loader',
+            options: {
+              ident: 'postcss',
+              plugins() {
+                return [
+                  Autoprefixer({
+                    browsers: [
+                      'ie >= 10',
+                      'last 2 versions',
+                      'last 2 android versions',
+                      'last 2 and_chr versions',
+                      'iOS >= 8',
+                    ],
+                  }),
+                  CustomProperties({
+                    warnings: false,
+                  }),
+                ];
               },
             },
-            {
-              loader: 'less-loader',
-              options: {
-                modifyVars: theme,
-              },
+          },
+          {
+            loader: 'less-loader',
```

Comment:
Are we still using less?

Reply:
Ant Design is using less unfortunately

Reply:
![](https://media.giphy.com/media/gCfHsvAQHtdcs/giphy.gif)